engine/table/src/main/java/io/deephaven/engine/table/impl/TableUpdateValidator.java

```diff
@@ -212,7 +216,13 @@ private void onUpdate(final TableUpdate upstream) {
                 }
                 result.append("\n");
                 resultTable.notifyListenersOnError(new RuntimeException(result.toString()), null);
+                return;
             }
+
+            final TableUpdateImpl downstream = TableUpdateImpl.copy(upstream);
+            downstream.modifiedColumnSet = resultTable.getModifiedColumnSetForUpdates();
+            transformer.transform(upstream.modifiedColumnSet(), downstream.modifiedColumnSet);
```

Comment:
I believe this to be unnecessary. The source table cannot safely mutate its reusable MCS until the next cycle.